src/org/labkey/test/tests/SampleTypeFolderExportImportTest.java

```diff
@@ -150,13 +150,12 @@ protected boolean areDataListEqual(List<Map<String, String>> list01, List<Map<St
             {
                 if(logMismatch)
                 {
-                    StringBuilder errorMsg = new StringBuilder();
-                    errorMsg.append("\n*************** ERROR ***************");
-                    errorMsg.append("\nFound a mismatch in the lists.");
-                    errorMsg.append("\nlist01(" + i + "): " + list01.get(i));
-                    errorMsg.append("\nlist02(" + i + "): " + list02.get(i));
-                    errorMsg.append("\n*************** ERROR ***************");
-                    log(errorMsg.toString());
+                    String errorMsg = "\n*************** ERROR ***************" +
+                            "\nFound a mismatch in the lists." +
+                            "\nlist01(" + i + "): " + list01.get(i) +
+                            "\nlist02(" + i + "): " + list02.get(i) +
+                            "\n*************** ERROR ***************";
+                    log(errorMsg);
```

Comment:
Not a suggestion for a change, just a note.
This "style" of error recording was done before the `checker()` was implemented. This, and other code like this, should be replaced with calls to `checker()`.